Makefile

```diff
@@ -368,14 +368,10 @@ ifeq ($(AUTODESPAWN), true)
 endif
 
 test-integration-%-cleanup:
-	${CONTAINER_TOOL}-compose -f scripts/integration/docker-compose.$*.yml rm --force --stop -v
+	cargo vdev integration stop $*
 
 test-integration-%:
-	RUST_VERSION=${RUST_VERSION} ${CONTAINER_TOOL}-compose -f scripts/integration/docker-compose.$*.yml build
-	RUST_VERSION=${RUST_VERSION} ${CONTAINER_TOOL}-compose -f scripts/integration/docker-compose.$*.yml run --rm runner
-ifeq ($(AUTODESPAWN), true)
-	make test-integration-$*-cleanup
```

Comment:
Is there any similar "cleanup" functionality in vdev ?
I'm actually not super familiar with that the existing make target does, but I suspect it might have something to do with removing stale containers, which is useful particularly while writing new tests and working through errors. At least, that is my vague recollection from some months ago.

Reply:
`cargo vdev int stop` 👍 or it's part of the `cargo vdev int test`

Reply:
The `cleanup` step is equivalent to the `stop` action, or at least it should be. I will restore that stanza.